query/src/interpreters/interpreter_query_log.rs

```diff
@@ -116,7 +116,7 @@ impl InterpreterQueryLog {
             Series::from_data(vec![event.query_kind.as_str()]),
             Series::from_data(vec![event.query_text.as_str()]),
             Series::from_data(vec![event.event_date as i32]),
-            Series::from_data(vec![event.event_time as u64]),
+            Series::from_data(vec![event.event_time as i64]),
```

Comment:
event.event_time is u64, but query_load table field type DateTime64Type maps to i64, is this safe?

Reply:
Why we change it to i64 here?

Reply:
Ok, I see. I think it's ok here